docs/get-started/yarn-gpu.md

```diff
@@ -117,3 +117,41 @@ On all masters:
 ```bash
 sudo systemctl restart hadoop-yarn-resourcemanager.service
 ```
+
+Note: If `cgroup` is mounted on `tmpfs`, after one node rebooted,
+the cgroup directory permission get reverted, you can manually 
+update the cgroup permission:
+```bash
+chmod a+rwx -R /sys/fs/cgroup/cpu,cpuacct
+chmod a+rwx -R /sys/fs/cgroup/devices
+```
+or you can add the operation as init.d scripts:
+Create a cgroup.mount script
+```bash
+sudo vim /etc/systemd/system/mountCgroup.service
+sudo chmod 777 /etc/systemd/system/mountCgroup.service
```

Comment:
Does  /etc/systemd/system/mountCgroup.service need to have 777 permissions, or 664?

Reply:
updated, thanks